fretboard/src/main/java/mozilla/components/service/fretboard/Fretboard.kt

```diff
@@ -43,4 +47,17 @@ class Fretboard(
             // Keep using the local experiments
         }
     }
+
+    /**
+     * Checks if the user is part of
+     * the specified experiment
```

Comment:
nit: I wonder why you break your javadoc comments so early in a lot of cases. It's fine to use all available 120 characters for the comments 🙃